notepadqq.pro

```diff
@@ -1,4 +1,4 @@
 TEMPLATE = subdirs
-SUBDIRS = src/ui \
-    src/ui-tests
+SUBDIRS = src/ui
+#    src/ui-tests
```

Comment:
Does this actually break anything? I still get differentIndendation warnings in tabs.